sys/kern/subr_bus.c

```diff
@@ -5108,7 +5109,6 @@ root_bus_module_handler(module_t mod, int what, void* arg)
 {
 	switch (what) {
 	case MOD_LOAD:
-		TAILQ_INIT(&bus_data_devices);
```

Comment:
This change doesn't move the needle enough to take at this time. bus_data_devices is the first thing that does this, so that makes this a little inconsistent. It's fine how it is, and there's not a compelling reason to  do this. Please drop this change from this commit series.

Reply:
I has been considering squashing it into the move to => `SI_SUB_INTR`.  Mainly I had looking at what could be pulled out of `root_bus_module_handler()`.